### PR TITLE
Civ 1395 further changes to error message

### PR DESCRIPTION
**Pull request by TatsyMaslova**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-1395

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/views/features/response/citizenPhoneNumber/citizen-phone.njk [hunk 1]
```diff
@@ -22,12 +23,18 @@
...
-                     value="{{ form.telephoneNumber }}">
+              {{ govukInput({
+                id: "telephoneNumber",
+                name: "telephoneNumber",
+                value: form.telephoneNumber,
+                hint: {
+                  text: "Use numbers only, for example, 01632960001."
+                },
+                errorMessage: {
+                  html: t(errors[0].text)
```
Comment (sampankumar) [inline src/main/views/features/response/citizenPhoneNumber/citizen-phone.njk:34]: Are we only expecting error in first index? Or do we need to iterate the errors?

Comment (TatsyMaslova) [reply src/main/views/features/response/citizenPhoneNumber/citizen-phone.njk:34]: yes. we only have one error message for this field.

**Review by sampankumar — APPROVED** [review]

**Review by osheaf01 — COMMENTED** [review]
LGTM

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**